conf/compose/docker-compose.cms.yml

```diff
@@ -3,8 +3,12 @@ services:
   cms:
     image: ${CMS_IMAGE}:${CMS_TAG}
     volumes:
-      - ${CAMINO_HOME}/conf/cms/secrets.py:/code/taccsite_cms/secrets.py
+      - ${CAMINO_HOME}/conf/cms/secrets.py:/code/taccsite_cms/secrets.py:ro
       - ${CAMINO_HOME}/conf/uwsgi/uwsgi_cms.ini:/code/uwsgi.ini
+      - ${CAMINO_HOME}/conf/nginx/nginx.conf:/etc/nginx/nginx.conf
```

Comment:
These could be dropped from the deployments repo override files for the CMS containers after being added here. They are always the same.